src/core/jbig2.js

```diff
@@ -14,6 +14,12 @@
  */
 
 import {
+  CCITTFaxStream, DecodeStream, Stream
+} from './stream';
+import {
+  Dict, isDict, isStream
+} from './primitives';
+import {
```

Comment:
Nit: It looks like all of these `import` statements would fit on a single line, i.e.
```javascript
import { CCITTFaxStream, DecodeStream, Stream } from './stream';
import { Dict, isDict, isStream } from './primitives';
import { log2, readInt8, readUint16, readUint32, shadow } from '../shared/util';
```